layouts/_default/baseof.html

```diff
@@ -28,11 +28,43 @@
 			<div class="container container-inner">
 				<div class="logo" role="banner">
 					<a class="logo__link" href="{{ "" | relLangURL }}" title="{{ .Site.Title }}" rel="home">
-						<div class="logo__title">{{ .Site.Title }}</div>
-						{{ with .Site.Params.subtitle }}<div class="logo__tagline">{{ . }}</div>{{ end }}
+						<div class="logo__left">
```

Comment:
Don't mix with #66

Reply:
I'm getting a bit confused about all these branches, to be honest.